### PR TITLE
#159403248 Increase coverage by 75%

### PR DESCRIPTION
**Pull request by veeqtor**
#### What does this PR do?

Increase coverage by 75%

#### Description of Task to be completed?

- add tests for user profile modifications
- modify a utility function
- modify validate.js

#### How should this be manually tested?

- After cloning the repo, `cd` into the directory
- Run  `cp .env.sample .env`
- Fill in the required database fields
- Run `npm install`
- Run `npm run migrate`
- Run `npm test`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159403248](https://www.pivotaltracker.com/story/show/159403248)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: helpers/validate.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class ValidateUser {
   static validatePassword(str) {
     const passwordFormat = /((^[0-9]+[a-z]+)|(^[a-z]+[0-9]+))+[0-9a-z]+$/i;
     if (!str || str.trim().length < 1) {
-      return { error: 'Password cannnot be empty' };
+      return { error: 'Password cannot be empty' };
```
Comment (klevamane) [inline helpers/validate.js:48]: Implement returning 'error' as stated in the readme.

**Review by klevamane — CHANGES_REQUESTED** [review]
Follow the method on the Readme when returning a response error.

File: helpers/validate.js (hunk 1 above)
Comment (anneKay) [reply helpers/validate.js:48]: I believe this is a function I wrote. This return is from a helper method in this class and not from the json response. The Json response using this method returns errors in the right format.

**Review by OKiMaureen — APPROVED** [review]

**Review by klevamane — APPROVED** [review]
I understand that the return format is from a helper method in this class and not from the json response.

**Review by anuonifade — APPROVED** [review]
LGTM

**Pull request merged**